packages/cli/src/commands/server/middleware/openURLMiddleware.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import launchChrome from '../launchChrome';
```

Comment:
I would advise to not use `launchChrome` as it only launches chrome and not the default browser.

Reply:
+1 for this (as a Safari user) - thanks for this PR though!

Reply:
Ah, I am also a safari user heh. I will make a change for this.

Reply:
I think you can use [`opn`](https://github.com/react-native-community/cli/blob/master/packages/cli/src/commands/server/launchChrome.js#L11) directly with the URL and it _should_ just work

Reply:
@orta correct! that is what i did.